samples/dynamically_add_module_sample/src/main.c

```diff
@@ -11,49 +11,102 @@ int main(int argc, char** argv)
     if (argc != 3)
     {
         printf("usage: dynamically_add_module_sample modulesConfigFile linksConfigFile\n");
-        printf("where configFile is the name of the file that contains the Gateway configuration\n");
+        printf("where modulesConfigFile is the name of the file that contains gateway modules and linksConfigFile is the name of the file that contains gateway links.\n");
+        printf("press ENTER to exit.\n");
```

Comment:
This code:
```c
    printf("press ENTER to exit.\n");
    (void)getchar();
```

...should only be run in the **success** case; you can remove it from all the failure cases.